leaderelection/leader_election.go

```diff
@@ -40,7 +40,7 @@ const (
 	defaultRenewDeadline = 10 * time.Second
 	defaultRetryPeriod   = 5 * time.Second
 
-	defaultReleaseOnCancel = false
+	defaultReleaseOnCancel = true
```

Comment:
any specific reason for this change?

Reply:
Please do not change the default, it would change the default behavior of all sidecars. 
There is some discussion and open items in https://github.com/kubernetes-csi/external-attacher/pull/633 that need to be resolved first.